druid/examples/text.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Druid Authors.
```

Comment:
Following Google practice: current year for new files.

```suggestion
// Copyright 2020 The Druid Authors.
```